app/src/main/res/layout-land/administrator_controls_download_permissions_view.xml

```diff
@@ -34,6 +34,8 @@
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:background="@color/white"
+      android:focusable="true"
+      android:onClick="@{(v) -> viewModel.onTopicWifiUpdatePermissionChanged()}"
```

Comment:
Earlier we were using `android:onCheckedChanged ` in `SwitchCompat`.  Because of this once the user double-taps to toggle the switch the reader was unable to read the new state. For example: `ON, Switch, Double tap to toggle` -> **Double tap action by learner**. No output.
But removing it and replacing it with `onClick` on double tap the reader does read the new state.